lib/pages/editor-page.js

```diff
@@ -104,7 +104,7 @@ export default class EditorPage extends BaseContainer {
 
 	dismissMediaModal() {
 		const driver = this.driver;
-		const cancelSelector = webdriver.By.xpath( "//div[contains(@class, 'editor-media-modal')]//span[contains(@class, 'dialog__button-label') and contains(text(), 'Cancel' )]" );
+		const cancelSelector = webdriver.By.css( 'div.dialog__action-buttons > button:nth-child(1) > span' );
```

Comment:
Yeah, I'd vote either add a data attribute or use `div.dialog__action-buttons button:not(.is-primary)` over the nth-child() bit